server/routes/changeLocation/handlers/changeLocation.ts

```diff
@@ -0,0 +1,20 @@
+import { Request, Response } from 'express'
+import UserService from '../../../services/userService'
+
+export default class ChangeLocationRoutes {
+  constructor(private readonly userService: UserService) {}
+
+  GET = async (req: Request, res: Response): Promise<void> => {
+    const { user } = res.locals
+    const prisonCaseloadFromNomis = await this.userService.getPrisonUserCaseloads(user)
+    const caseload = prisonCaseloadFromNomis.map(c => ({ value: c.caseLoadId, text: c.description }))
```

Comment:
meh.. this could be pushed through as is and then filtered with  a custom filter into a list but NBD.

We have generic filters in one or more projects like: `prisons | toOptions('caseLoadId', 'description')`
 where the value is determined by the first key and text is determined by second key

Reply:
may be